src/qtism/runtime/results/AssessmentResultBuilder.php

```diff
@@ -39,23 +43,27 @@
  */
 class AssessmentResultBuilder extends AbstractResultBuilder
 {
-
+    const CUSTOM_PARAMETERS_SESSION_IDENTIFIER = 'LtiCustomParameters';
+    const CUSTOM_PARAMETERS_URI = 'http://lti-custom-parameter/';
+    
     /**
      * Build Result
      *
+     * @param array $customParameters LTI custom parameters to be inserted into the results' context.
+     *
      * @return AssessmentResult
      */
-    public function buildResult()
+    public function buildResult(array $ltiCustomParameters = null)
```

Comment:
imo `array $ltiCustomParameters = []` is safer, dueto the private buildContext method expetcation